gits/test/capabilities_test.py

```diff
@@ -323,20 +323,42 @@ def test_cap_dch(self):
         want = s[:term._cur_x] + s[term._cur_x + 1:]
         self._check_string(want, (0, 0), (len(want), 0))
 
-    @unittest.skip('skip')
-    def test_cap_dl1(self):
-        """The terminal should have the possibility to delete a line. """
+    def test_cap_dl(self):
+        """The terminal should have the possibility to delete ``n`` number of
+        lines.
+        """
         term = self._terminal
-
-        # The y position of the cursor is on the first line.
-        self._check_cap_dl1(['s'] * term._right_most, (0, 1))
-
-        # The y position of the cursor is on the last line.
-        self._check_cap_dl1(['s'] * term._right_most, (0, term._bottom_most))
-
-        # The y position of the cursor is on an arbitrary line.
-        rand_y = random.randint(1, term._bottom_most - 1)
-        self._check_cap_dl1(['s'] * term._right_most, (0, rand_y))
+        self._check_cap_dl(0, [
+            (['f'] * term._right_most, (0, 0)),
+        ])
```

Comment:
keep the code on one line

`self._check_cap_dl(0, [(['f'] * term._right_most, (0, 0))])`